owncloudDomain/src/main/java/com/owncloud/android/domain/files/usecases/SaveFileUseCase.kt

```diff
@@ -0,0 +1,32 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Christian Schabesberger
+ * Copyright (C) 2020 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.files.usecases
+
+import com.owncloud.android.domain.BaseUseCaseWithResult
+import com.owncloud.android.domain.files.FileRepository
+import com.owncloud.android.domain.files.model.OCFile
+
+class SaveFileOrFolderUseCase(
```

Comment:
File name and class name are different. Rename file to SaveFileOrFolderUseCase or rename class, your choice 👍

Reply:
done